content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -64,31 +61,76 @@ To combine an event, click the three dots next to an event and select **+ Combin
 
   **Note**: Only completed RUM sessions contribute to conversion rates.
 
-- **Time to convert** - The time it took for the user to complete the step events.
+- **Time to convert**: The time it took for the user to complete the step events.
 
 You can measure these attributes **across all steps** or between **specific steps**.
 
 {{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_conversion.png" alt="Measure attributes across all steps or specific steps." style="width:60%;" >}}
 
-Use the **filter** selector to filter by various criteria that you define.
+Use the **filter** selector to filter by various criteria that you define. These filters are applied to all steps of the funnel.
 
 Next, click a datapoint to **investigate the specific attributes** that might have affected conversion rates, such as page load speed, ease of navigation, or checkout experience.
 
+## How Datadog computes conversion metrics
+
+Consider a funnel with events `A → B → C` and event steps: 
+
+```
+A, A, A, B, C, C
+```
+
+If a user performs the actions as in the example above, Datadog counts it as one conversion. This is because the conversion calculations only look at the first element **A** matched and the first element **C** matched. 
+
+<div class="alert alert-info">
+If the user performs the event steps <code>A, A, A, B, C, C, A,...</code>, it would count as two conversions. The first conversion ends at <code>A, A, A, B, C, C</code>, and the second conversion begins with <code>A,...</code>
+
+</div>
+
+
+{{< img src="product_analytics/journeys/funnel_analysis/funnel_analysis_conversion.png" alt="Select a conversion measure, whether Unique or Total, to determine how your session conversions are counted." style="width:80%;" >}}
+
+
+**Unique**: This means that your conversion (session, user, or account) is counted only once per session, user or account. So, if the user performs the actions A → B → C → A → B → C during the session or time frame, it counts as 1 conversion.
+
+**Total**: This means that your conversion (session, user, or account) is each time the user completes the defined funnel. For example, if the user performs the actions A → B → C → A → B → C during the session or time frame, it counts as 2 conversions.
+
+
+If you analyze your funnel by `user` or by `account`, you can define your conversion timeframe in hours or days of the first event. The default timeframe for conversions is 1 day (a 24-hour window not calendar dates) to determine if a conversion happened or not.
+ 
+The average time between steps is calculated by looking at the average duration between each conversion and takes all conversions into account, no matter if `unique` or `total` is selected for the counts.
+
+
+The **conversion rate** compares the total number of conversions to the total number of users, accounts, or sessions that enter the funnel.
```

Comment:
```suggestion
```
Looks like we explain this in the previous section, so I think you're off the hook explaining it again here!